pkg/env/core.go

```diff
@@ -72,3 +72,13 @@ func NewCoreForCI(ctx context.Context, log *logrus.Entry) (Core, error) {
 		isLocalDevelopmentMode: isLocalDevelopmentMode,
 	}, nil
 }
+
+// NewMockCore returns an env.Core which does nothing, but allow for
+// tests to be written without any environment
+func NewMockCore(hostname, tenant, subscriptionID, location, resourceGroup, storageEndpointSuffix string) Core {
```

Comment:
NewMockCore is a sensible choice for the given purpose. Now when we look at the location, NewEmptyCore could work even better, as in ARO-RP we try to put mocks in a separate folders. 

Other option, which will fit the current folder design is placing the mock in `pkg/util/mocks` amongst other mocks.